### ISSUE TITLE
pr?

### ISSUE THREAD

**Issue by vyrus001** [wheelerlaw/go-mimikatz#2]
any chance of getting a pr? (nice work!)

**Comment by wheelerlaw** [comment]
@vyrus001 Yes, absolutely! It is a little bit different than your original project (it's a fork of a fork), but I would be happy to make a PR for it. I am going to do a little bit of work getting the Travis build working and then I will publish a PR.
